apps/changelog/classes/log_entry.py

```diff
@@ -83,8 +82,7 @@ def release_version(self):
                             sha=m.group('sha'),
                             branch=m.group('branch'),
                         )
-                    except Exception as ex:
-                        print(ex)
+                    except Exception:
```

Comment:
Not in this PR, but can we also fix a spelling error in `regex.py`

`htk/apps/changelog/constants/regexes.py`
- `# never format: 'tag: deploy-202211031730-4acb099a93-master'`
- Should read `newer` instead of `never`

Reply:
Done